mapgwm/tests/test_headobs.py

```diff
@@ -4,22 +4,23 @@
 
 def test_preprocess_headobs(test_output_folder):
     # input files
-    path = 'mapgwm/tests/data/headobs'
```

Comment:
Looking at the log, it looks like the build is failing on some pathing issues. I just added another path fixture for the test data folder (see [the current headobs test](https://github.com/aleaf/usgs-map-gwmodels/blob/master/mapgwm/tests/test_headobs.py)). Would probably be easiest/most robust to use that, similar to the current test.